tests/infra/network.py

```diff
@@ -154,6 +154,34 @@ def _adjust_local_node_ids(self, primary):
             for n in self.nodes:
                 n.node_id = n.node_id + first_node_id
 
+    def wait_for_snapshot_evidence_commit_proof(self, node, snapshot, timeout=3):
```

Comment:
This is a little awkward but I believe the easiest way to programmatically test that the commit of the evidence of given snapshot is covered by a signature in the ledger.